Makefile

```diff
@@ -70,9 +70,6 @@ build-docs:
 	-mkdir docs
 	cp -R src/biokbase-doc/_build/html/* docs/
 
-docker-base:
-	docker build -t kbase/narrbase:3.0 base/
-
 docker-narrative:
```

Comment:
`docker-narrative` is out of date and probably not used either, might get nuked next time.